x-pack/plugins/fleet/public/components/package_icon.tsx

```diff
@@ -24,6 +24,8 @@ export const CardIcon: React.FunctionComponent<UsePackageIconType & Omit<EuiIcon
   const { icons } = props;
   if (icons && icons.length === 1 && icons[0].type === 'eui') {
     return <EuiIcon size={'xl'} type={icons[0].src} />;
+  } else if (icons && icons.length === 1 && icons[0].type === 'svg') {
+    return <EuiIcon size={'xl'} type={icons[0].src} />;
```

Comment:
Should merge https://github.com/elastic/kibana/pull/113200 first, so we do not need to introduce this change.